Blockzilla/Onboarding/TooltipTableViewCell.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TooltipTableViewCell.swift
+//  Blockzilla
+//
+//  Created by catalin.neculaide on 25.02.2022.
+//  Copyright © 2022 Mozilla. All rights reserved.
+//
+
+import UIKit
+
+class TooltipTableViewCell: UITableViewCell {
+
+    var tooltip = TooltipView()
```

Comment:
private

Reply:
cant make i private, I'm setting the delegate inside the cell